### ISSUE TITLE
Inherited Failure on beta engine 

### ISSUE THREAD

**Issue by DineshDhyani** [opengeospatial/teamengine#96]
I am running wms 1.1.1 tests. I tried to replicate locally using 4.1.0 version (and tag 4.1.0b) and was successfully able to run tests without any inherited failure. However when I am trying to run it against Beta engine of OGC which has same version deployed it gives Result: Failed (InheritedFailure).

Not sure what the problem could be. Can anyone please help? I am using latest CTL file available.

**Comment by bermud** [comment]
Do you have a public endpoint that we can test. Thank you!

**Comment by DineshDhyani** [comment]
http://182.71.43.203:8080/rest/Spatial/WMS?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.1.1  is the capability URL used against the public machine. Thanks.

**Comment by bermud** [comment]
![pic 2015-04-29 at 6 18 54 pm](https://cloud.githubusercontent.com/assets/3738898/7402660/3bbee0cc-ee9c-11e4-9a71-a3ef403841b6.jpg)
![pic 2015-04-29 at 6 18 48 pm](https://cloud.githubusercontent.com/assets/3738898/7402661/3bc11da6-ee9c-11e4-8145-5de8926eed8e.jpg)

**Comment by DineshDhyani** [comment]
Thanks Luis!.

**Comment by bermud** [comment]
@DineshDhyani,  the link is not working any more. Can you please make it available again.Thank you.

**Comment by DineshDhyani** [comment]
@bermud: Link should be working now. Apologies for disruption. Thanks.

**Comment by bermud** [comment]
The issue was due to  a memory leak in the server. Restarting tomcat fixed the problem. Please try again http://cite.opengeospatial.org/te2/

```

Fatal Error: Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.MalformedURLException: no protocol: ?SeRvIcE=WMS&BoGuS=ignored&ReQuEsT=GetCapabilities&VeRsIoN=1.1.1; SystemID: file:/srv/local/teamengine-te2/TE_BASE/work/srv_local_teamengine-te2_TE_BASE_scripts_wms_1.1.1_ctl_/wms/wms$basic_elements-param_rules-order_and_case-3.test; Line#: 54; Column#: -1
Fatal Error: Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.MalformedURLException: no protocol: ?SeRvIcE=WMS&ReQuEsT=GetCapabilities&VeRsIoN=1.1.1; SystemID: file:/srv/local/teamengine-te2/TE_BASE/work/srv_local_teamengine-te2_TE_BASE_scripts_wms_1.1.1_ctl_/wms/wms$basic_elements-version-negotiation-2.test; Line#: 53; Column#: -1
Fatal Error: Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.MalformedURLException: no protocol: ?SeRvIcE=WMS&VeRsIoN=0.0.0&ReQuEsT=GetCapabilities; SystemID: file:/srv/local/teamengine-te2/TE_BASE/work/srv_local_teamengine-te2_TE_BASE_scripts_wms_1.1.1_ctl_/wms/wms$basic_elements-version-negotiation-4.test; Line#: 50; Column#: -1
Fatal Error: Error in call to extension function {public synchronized void com.occamlab.te.TECore.callTest(net.sf.saxon.expr.XPathContext,java.lang.String,java.lang.String,net.sf.saxon.om.NodeInfo,java.lang.String) throws java.lang.Exception}: Exception in extension function java.util.NoSuchElementException; SystemID: file:/srv/local/teamengine-te2/TE_BASE/work/srv_local_teamengine-te2_TE_BASE_scripts_wms_1.1.1_ctl_/wms/wms$wms_main.test; Line#: 1069; Column#: -1
See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0060/log.xml
See the detail error Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0060/error_log/log.txt
See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0060/log.xml
Warning: The child axis starting at an attribute node will never select anything
Warning: The child axis starting at an attribute node will never select anything
Warning: The child axis starting at an attribute node will never select anything
See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0060/log.xml
Warning: The child axis starting at an attribute node will never select anything
Warning: The child axis starting at an attribute node will never select anything
Warning: The child axis starting at an attribute node will never select anything
See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0060/log.xml
See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0060/log.xml
Warning: The child axis starting at an attribute node will never select anything
Warning: The child axis starting at an attribute node will never select anything
Warning: The child axis starting at an attribute node will never select anything
See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0060/log.xml
See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/mideveloper/s0062/log.xml
Apr 30, 2015 9:59:27 AM org.apache.catalina.loader.WebappClassLoader modified
SEVERE:     Resource '/WEB-INF/classes/logging.properties' is missing
Apr 30, 2015 9:59:27 AM org.apache.catalina.core.StandardContext reload
INFO: Reloading Context with name [/te2] has started
Apr 30, 2015 9:59:27 AM org.apache.catalina.loader.WebappClassLoader clearReferencesThreads
SEVERE: The web application [/te2] appears to have started a thread named [Thread-13] but has failed to stop it. This is very likely to create a memory leak.
Apr 30, 2015 9:59:38 AM org.apache.catalina.core.StandardContext reload


```

**Comment by DineshDhyani** [comment]
Thanks Luis. I can successfully run all test cases on beta site now.

we still have 2 test cases failing for certification link though....do we know when 4.1.0 will be deployed and can be used for certification?

**Comment by DineshDhyani** [comment]
I am again facing this issue. Please refer  s0067 on beta site, inherited failure occuring  again.

**Comment by bermud** [comment]
This is a different problem. We are investigating. Thank you for reporting.

```
May 05, 2015 4:43:44 AM com.occamlab.te.web.TestServlet process
INFO: Creating test session in /srv/local/teamengine-te2/TE_BASE/users/IGMueTo
Fatal Error: Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.MalformedURLException: no protocol: ; SystemID: file:/srv/local/teamengine-te2/TE_BASE/work/srv_local_teamengine-te2_TE_BASE_scripts_wms_1.1.1_ctl_/wms/wms$wms_main.test; Line#: 77; Column#: -1
May 05, 2015 4:46:10 AM com.occamlab.te.TECore executeTest
SEVERE: Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.MalformedURLException: no protocol: 
			See the detail test Report /srv/local/teamengine-te2/TE_BASE/users/IGMueTo/s0006/log.xml
			See the detail error Report /srv/local/teamengine-te2/TE_BASE/users/IGMueTo/s0006/error_log/log.txt
May 05, 2015 4:46:42 AM com.occamlab.te.web.TestServlet process
INFO: Creating test session in /srv/local/teamengine-te2/TE_BASE/users/IGMueTo
May 05, 2015 4:50:48 AM com.occamlab.te.TECore parse
SEVERE: Error invoking parser {http://www.occamlab.com/te/parsers}XMLValidatingParser
java.lang.NullPointerException
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.occamlab.te.TECore.parse(TECore.java:2007)
	at com.occamlab.te.TECore.parse(TECore.java:1923)
```

**Comment by DineshDhyani** [comment]
Hi Luis, 
Is this issue fixed in the latest release of teamengine ? 

Facing below 2 issues now for 4.0.5 version. 
Request d1e11686_1:
   Method: get
   URL: http://182.71.43.203:8080/rest/Spatial/WMS?&LaYeRs=cite%3ABridges&SrS=EPSG%3A4326&BbOx=0,-0.0020,0.0040,0&WiDtH=100&ReQuEsT=GetMap&StYlEs=&HeIgHt=100&FoRmAt=image%2Fjpeg&VeRsIoN=1.1.1
Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.lang.reflect.InvocationTargetException
Result: Failed


Request d1e9283_1:
   Method: get
   URL: http://182.71.43.203:8080/rest/Spatial/WMS?&SrS=EPSG:4326&LaYeRs=NonExistant&StYlEs=&ReQuEsT=GetMap&HeIgHt=96&FoRmAt=image/jpeg&VeRsIoN=1.1.1&BbOx=-0.0042,-0.0024,0.0042,0.0024&WiDtH=168
Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.ConnectException: Connection timed out
Result: Failed

**Comment by bermud** [comment]
Fix in 38c9956730aa4bcae4021074b86143560931fe74
